docs/variables.md

```diff
@@ -661,6 +684,20 @@ Turns on RocksDB memory-charging related features (BlockBasedTableOptions::cache
 
 This variable is disabled (OFF) by default.
 
+### `rocksdb_check_iterate_bounds`
+
+| Option       | Description             |
+|--------------|-------------------------|
+| Command-line | --rocksdb-check-iterate-bounds |
+| Dynamic      | Yes                     |
+| Scope        | Global, Session         |
+| Data type    | Boolean                 |
+| Default      | ON                      |
+
+This variable has been implemented in [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md).
+
+This variable checks the upper and lower bounds of an iterator in RocksDB. The default value in `ON` which means this variable is enabled.
```

Comment:
This variable enables checking of rocksdb iterator upper/lower bounds during iterating

Reply:
fixed